build-package.ps1

```diff
@@ -9,5 +9,11 @@ Import-Module "$PSScriptRoot/scripts/ps-modules/Build" -Force -DisableNameChecki
 Write-Banner -Level 1 -Title "Installing preconfigured package: `"$PackageName`""
 $pkg = Get-PackageInfo -PackageName $PackageName
 
+if ($pkg -eq $null) {
+    $selectedSection = if ((Get-IsOnWindowsOS)) { "win" } elseif ((Get-IsOnMacOS)) { "mac" } else { "linux" }
```

Comment:
It might be worth extracting this as something like `Get-OSType` or something like that, so we can reuse it here and in Get-PackageInfo?

Reply:
I wondered about that as well.  [98c445f](https://github.com/TechSmith/ThirdParty-Packages-vcpkg/pull/45/commits/98c445f0a8bd86859e6a6b1a06b4c026aebe6129)